### PR TITLE
Add license comments checker.

### LINKED ISSUE #724: [CI] License Style check
**Summary**
1. New contributors always forget to add the license comments in creating new rust file.
2. Vscode and Clion have the feature to auto-import used imports. This will break the file style. 

Better to make a check for this.
```
use common_exception::ErrorCodes;
use common_exception::Result;

// Copyright 2020-2021 The Datafuse Authors.
```

### PR DESCRIPTION
**Pull request by jovany-wang**
## Summary

Add a github action to check license comments.

## Related Issues

Fixes #724

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

**Comment by jovany-wang** [comment]
@sundy-li Could you approve the CI workflows?

**Comment by drmingdrmer** [comment]
Coooooooool

**Review by sundy-li — APPROVED** [review]
Looks great to me. Thanks for your contribution @jovany-wang

File: fusequery/query/src/pipelines/transforms/transform_remote_test.rs [hunk 1]
```diff
@@ -1,8 +1,8 @@
+// Copyright 2020-2021 The Datafuse Authors.
+//
+// SPDX-License-Identifier: Apache-2.0.
+
 // <<<<<<< HEAD
```
Comment (drmingdrmer) [inline fusequery/query/src/pipelines/transforms/transform_remote_test.rs:5]: Is this an unresolved git conflict?

Comment (BohuTANG) [reply fusequery/query/src/pipelines/transforms/transform_remote_test.rs:5]: This is under design by @zhang2014  
It will be fixed later

Comment (jovany-wang) [reply fusequery/query/src/pipelines/transforms/transform_remote_test.rs:5]: Yes. Do you know if this file is unused? If it's unused any longer, let's remove it.

Comment (jovany-wang) [reply fusequery/query/src/pipelines/transforms/transform_remote_test.rs:5]: It's better to leave a TODO comment on unaddressed issue. :)

**Pull request merged**